scipy/io/_netcdf.py

```diff
@@ -178,8 +178,8 @@ class netcdf_file:
     --------
     To create a NetCDF file:
 
-    >>> from scipy.io import netcdf
```

Comment:
This example is a little unfortunate. It did encourage people to use the `netcdf` namespace.